test/snippet/io/alignment_file/alignment_file_output_input_range.cpp

```diff
@@ -0,0 +1,39 @@
+#include <tuple>
+#include <string>
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/io/alignment_file/all.hpp>
+#include <seqan3/std/filesystem>
+
+int main()
+{
+    auto tmp_dir = std::filesystem::temp_directory_path();
+
+    // Create a /tmp/input.sam for the snippet below
```

Comment:
We have this scope of creating a file for the actual snippet a lot of times... Disadvantages: It is copy-pasted code and also distracts from the really important snippet part. But I don't see a way to avoid this, if we want executable snippets.